Src/Particle/AMReX_StructOfArrays.H

```diff
@@ -52,6 +53,28 @@ struct StructOfArrays {
     /** Get access to the particle Int Arrays (only compile-time components) */
     [[nodiscard]] const std::array< IntVector,  NInt>& GetIntData  () const { return m_idata; }
 
+    /** Get the names for the real SoA components **/
+    [[nodiscard]] std::vector<std::string> GetRealNames () const
+    {
+        if (m_rdata_names) {
+            return *m_rdata_names;
+        }
+        else {
+            throw std::runtime_error("SoA: no Real names defined!");
```

Comment:
We could also return:
```suggestion
            return std::vector<std::string>();
```

Reply:
I think return is better.